example/synchronous_lstm/imdb.py

```diff
@@ -76,7 +76,7 @@ def get_dataset_file(dataset, default_dataset, origin):
     return dataset
 
 
-def load_data(path="/Tmp/mastropo/imdb.pkl", n_words=100000, valid_portion=0.1, maxlen=None,
+def load_data(path="/tmp/imdb.pkl", n_words=100000, valid_portion=0.1, maxlen=None,
```

Comment:
The path i the non-synchrone lstm is "imdb.pkl" I would reuse the same. I just pushed a commit to this PR that does this.